.github/workflows/build_all_apps.yml

```diff
@@ -0,0 +1,115 @@
+name: Build all apps on latest SDK
+
+on:
+  workflow_dispatch:
+    inputs:
+      sdk_branch:
+        type: string
+        required: false
+        default: ''
+  pull_request:
+
+jobs:
+  prepare-matrix:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

Comment:
v4?

Reply:
I would tend to prefer keeping it like this as it make it the same as on master branch.